### DISCUSSION TITLE
push error which LFS

### DISCUSSION THREAD

**Discussion by openvideogame** [community/community#112200] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Using my team github, when I try to push my commit, this error happened

```
> batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
> Uploading LFS objects:   0% (0/60), 0 B | 0 B/s, done.
> error: failed to push some refs to "(adress)"
```

so I confirm my team Github LFS, bandwidth is used 1.74GB, but in display, our files use 0 byte.

my file which I wanna push size is only 100MB. but I cannot push which this error.
what is this error? how to fix it?

Please tell me the solution.

![thumbnail_image](https://github.com/community/community/assets/126433429/b5c9b88c-06ac-4c1e-81de-80c5ad861d1b)

**Comment by SonmxNota** [comment]
I have the same problem but no effective solution yet
